gateway-ha/src/main/resources/mysql/V2__add_routingGroup_to_query_history.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE query_history
+    ADD routing_group VARCHAR(255);
```

Comment:
nit: end the file with a newline. This goes for the other `.sql` files as well

Reply:
sure!